playbooks/libvirt/openshift-cluster/launch.yml

```diff
@@ -10,6 +10,8 @@
     os_libvirt_storage_pool: "{{ libvirt_storage_pool | default('images') }}"
     os_libvirt_storage_pool_path: "{{ libvirt_storage_pool_path | default('/var/lib/libvirt/images') }}"
     os_libvirt_network: "{{ libvirt_network | default('default') }}"
+    os_libvirt_instance_memory: "{{ lookup('oo_option', 'libvirt_instance_memory') | default(1) }}"
```

Comment:
+1, I'll update to libvirt_instance_memory_mib, to allow for the more granular allocation